cicecore/cicedyn/general/ice_forcing.F90

```diff
@@ -5754,6 +5872,1576 @@ end subroutine init_snowtable
+                  Tsfc_bry(i,j,n,iblk) = c0;
+                  alvln_bry(i,j,n,iblk) = c0;     
+                  vlvln_bry(i,j,n,iblk) = c0;     
+                  apondn_bry(i,j,n,iblk) = c0;     
+                  hpondn_bry(i,j,n,iblk) = c0;     
+                  ipondn_bry(i,j,n,iblk) = c0;     
+                  !hbrine_bry(i,j,n,iblk) = c0;     
+                  !fbrine_bry(i,j,n,iblk) = c0;     
+                  iage_bry(i,j,n,iblk) = c0; 
+                  do k = 1,nilyr
+                     Tinz_bry(i,j,k,n,iblk) = c0;
+                     Sinz_bry(i,j,k,n,iblk) = c0;
+                  enddo  
+               enddo
+               uvel_bry(i,j,iblk) = c0; ! pedrocice
+               vvel_bry(i,j,iblk) = c0; ! pedrocice
+            enddo
+         enddo 
+      enddo
+
+      end subroutine init_forcing_bry
+
+!=======================================================================
+      subroutine boundary_files(yr)
+
+      
+      ! Construct filenames for sea-ice boundary data.
+      ! Edit for other directory structures or filenames.
+      ! Note: The year number in these filenames does not matter, because
+      !       subroutine file_year_bry will insert the correct year.
+      ! authors: Pedro Duarte, NPI
+
+      integer (kind=int_kind), intent(in) :: &
+           yr                   ! current forcing year
+
+      bry_file = &
+           trim(sea_ice_bry_dir)//'BRY_1996.nc'
+      call file_year_bry(bry_file,yr)
+
+!       if (my_task == master_task) then
+!         write (nu_diag,*) ' '
+!         write (nu_diag,*) 'Sea-ice boundary data year =', fyear
+!         write (nu_diag,*) 'Sea-ice boundary data file:'
+!         write (nu_diag,*) trim(bry_file)
+!          
+!       endif                     ! master_task
+
+      end subroutine boundary_files 
+!=======================================================================
+      subroutine file_year_bry (data_file, yr)
+
+! Construct the correct name of the sea-ice boundary file
+! to be read, given the year and assuming the naming convention
+! that filenames end with 'yyyy.nc'.
+
+      character (char_len_long), intent(inout) ::  data_file
+
+      integer (kind=int_kind), intent(in) :: yr
+
+      character (char_len_long) :: tmpname
+
+      integer (kind=int_kind) :: i
+      
+       
+
+      i = index(data_file,'.nc') - 5
+      tmpname = data_file
+      write(data_file,'(a,i4.4,a)') tmpname(1:i), yr, '.nc'
+      
+      
+!       if (my_task == master_task) then
+!       
+!         write (nu_diag,*) ' '
+!          write (nu_diag,*) 'file_year_bry : '
+!         write (nu_diag,*) 'Sea-ice boundary data year =', yr
+!         write (nu_diag,*) 'Sea-ice boundary data file:'
+!         write (nu_diag,*) trim(data_file)
+!          
+!       endif
+      
+      end subroutine file_year_bry
+
+!=======================================================================
+      subroutine get_forcing_bry
```

Comment:
@SmithJos13 Why do we need to have a routine to call another one without doing anything extra.